File: Examen3erPeriodo/EscuelaApi/schemas/Materia.js
module.exports = {
    idMateria:Number,
    foto:String,
    nombre:{
        type:String,
        hashKey:true
    },
    descripcion:String,
    dias:{
        type:Array,
        schema:[String]
    },
    horas:{
        type:Array,
        schema:[String]
    },
    matriculados:{
        type:Array,
        schema:[String]
    }
}

File: Examen3erPeriodo/appMaestro/app/src/main/java/com/example/appmaestro/domain/model/Materia.kt
package com.example.appmaestro.domain.model

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.squareup.moshi.JsonClass
import kotlinx.parcelize.Parcelize

@Parcelize
@Entity
@JsonClass(generateAdapter = true)
class Materia(
    @PrimaryKey(autoGenerate = false)
    val nombre : String ="",
    val idMateria : String="",
    val descripcion:String="",
    val foto : String="",
    val dias : List<String> = emptyList(),
    val horas : List<String> = emptyList(),
    val matriculados : List<String> = emptyList()
) : Parcelable { }